src/ci/docker/host-x86_64/i686-gnu/Dockerfile

```diff
@@ -25,4 +25,6 @@ RUN sh /scripts/sccache.sh
 
 ENV RUST_CONFIGURE_ARGS --build=i686-unknown-linux-gnu
 ARG SCRIPT_ARG
-ENV SCRIPT=${SCRIPT_ARG}
+COPY scripts/stage_2_test_set1.sh /scripts/
+COPY scripts/stage_2_test_set2.sh /scripts/
+ENV SCRIPT /scripts/${SCRIPT_ARG}
```

Comment:
Why the difference between `i686-gnu` and `i686-gnu-nopt`? Here is the `/scripts/` prefix, but not in the other one.

Reply:
because i686-gnu-nopt-2 needs to also run the stage 0 test, so we can't write the `/scripts` directory in the `ENV SCRIPT` of the dockerfile.

Reply:
Ok, let's keep it like this then.